tests/color_glyph_test.py

```diff
@@ -41,35 +43,81 @@ def _nsvg(filename):
 
 
 @pytest.mark.parametrize(
-    "view_box, upem, expected_transform",
+    "view_box, upem, width, ascender, descender, expected_transform, expected_width",
     [
         # same upem, flip y
-        ("0 0 1024 1024", 1024, Affine2D(1, 0, 0, -1, 0, 1024)),
+        ("0 0 1024 1024", 1024, 1024, 1024, 0, Affine2D(1, 0, 0, -1, 0, 1024), 1024),
         # noto emoji norm. scale, flip y
-        ("0 0 128 128", 1024, Affine2D(8, 0, 0, -8, 0, 1024)),
+        ("0 0 128 128", 1024, 1024, 1024, 0, Affine2D(8, 0, 0, -8, 0, 1024), 1024),
         # noto emoji emoji_u26be.svg viewBox. Scale, flip y and translate
-        ("-151 297 128 128", 1024, Affine2D(8, 0, 0, -8, 1208, 3400)),
+        (
+            "-151 297 128 128",
+            1024,
+            1024,
+            1024,
+            0,
+            Affine2D(8, 0, 0, -8, 1208, 3400),
+            1024,
+        ),
         # made up example. Scale, translate, flip y
         (
             "10 11 20 21",
             100,
-            Affine2D(a=5.0, b=0, c=0, d=-4.761905, e=-50.0, f=152.380952),
+            100,
+            100,
+            0,
+            Affine2D(a=4.761905, b=0, c=0, d=-4.761905, e=-47.738095, f=152.380952),
+            95,
+        ),
+        # noto emoji width, ascender, descender
+        (
+            "0 0 1024 1024",
+            1024,
+            1275,
+            950,
+            -250,
+            Affine2D(1.171875, 0, 0, -1.171875, 37.5, 950),
+            1275,
+        ),
+        # viewbox wider than tall
```

Comment:
suggest making these more descriptive, e.g. viewbox wider than tall so <what transform is expected>